install/debug.conf.template

```diff
@@ -1609,6 +1609,37 @@ ModPagespeedMessagesDomains Allow localhost
 </VirtualHost>
 
 <VirtualHost localhost:@@APACHE_SECONDARY_PORT@@>
+  ServerName redirecting-fetch.example.com
+  DocumentRoot "@@APACHE_DOC_ROOT@@"
+  ModPagespeedFileCachePath "@@MOD_PAGESPEED_CACHE@@"
+  ModPagespeedDomain redirecting-fetch.example.com:@@APACHE_SECONDARY_PORT@@
+  ModPagespeedMapOriginDomain 127.0.0.1:@@APACHE_SECONDARY_PORT@@ \
+    redirecting-fetch.example.com:@@APACHE_SECONDARY_PORT@@
```

Comment:
The incoming request will be for `redirecting-fetch.example.com`. We use something like `--proxy 127.0.0.1:@@APACHE_SECONDARY_PORT@@` in the tests.
I added the origin mapping here as an equivalent to that, to get the redirects to land where they should..